Inspicio/Views/Shared/_LoginPartial.cshtml

```diff
@@ -17,7 +17,7 @@
     <form asp-area="" asp-controller="Account" asp-action="Logout" method="post" id="logoutForm" class="navbar-right">
         <ul class="nav navbar-nav navbar-right"> 
             <li>
-                <a asp-area="" asp-controller="Manage" asp-action="Index" title="Manage"><img src="@User.GetProfilePicture(40)" />Hello <span class="main-user">@User.GetProfileName()</span></a>
+                <a asp-area="" asp-controller="Manage" asp-action="Index" title="Manage"><img style="height:40px;width:40px" src="@User.GetProfilePicture()" /> Hello <span class="main-user">@User.GetProfileName()</span></a>
```

Comment:
It would be better practice to set the styling (height, width etc..) in an external stylesheet rather than here.